admin/src/main/resources/io/buoyant/admin/js/router_client.js

```diff
@@ -83,17 +83,47 @@ var RouterClient = (function() {
     return summary;
   }
 
-  return function (metricsCollector, routers, client, $metricsEl, routerName, clientTemplate, metricPartial, $chartEl, colors) {
+  return function (metricsCollector, routers, client, $metricsEl, routerName, clientTemplate, metricPartial, $chartEl, colors, $toggleLinks, shouldExpandInitially) {
     template = clientTemplate;
     Handlebars.registerPartial('metricPartial', metricPartial);
     var clientColor = colors.color;
     var latencyLegend = createLatencyLegend(colors.colorFamily);
     var metricDefinitions = getMetricDefinitions(routerName, client.label);
 
+    var $expandLink = $toggleLinks.find(".client-expand");
+    var $collapseLink = $toggleLinks.find(".client-collapse");
+
     renderMetrics($metricsEl, client, [], [], clientColor);
-    var chart = SuccessRateGraph($chartEl, colors.color);
+    var chart = SuccessRateGraph($chartEl.find($(".client-success-rate")), colors.color);
+
+    // collapse client section by default (deal with large # of clients)
+    if(shouldExpandInitially) {
+      showClient();
+    } else {
+      toggleClientDisplay(false);
```

Comment:
Yep indeed it's a no-op, I just didn't want to go through the list of listeners when I already know we won't find the item.

Agreed that it would read more clearly in the if/else if it is just show vs hide.